src/Aspire.Hosting/Dashboard/DashboardViewModelService.cs

```diff
@@ -402,7 +402,8 @@ private ProjectViewModel ConvertToProjectViewModel(Executable executable)
             }
 
             var matchingService = _servicesMap.Values.SingleOrDefault(s => s.Metadata.Name == endpoint.Spec.ServiceName);
-            if (matchingService?.UsesHttpProtocol(out var uriScheme) == true)
+            if (string.Equals(matchingService?.Status?.State, "Ready", StringComparison.OrdinalIgnoreCase)
```

Comment:
I want to improve this kind of test. See https://github.com/dotnet/aspire/issues/1406. Just an FYI, it doesn't have to happen in this PR.

Reply:
Also, to add `StringComparers.ResourceState` so we use the same comparer across values.